backend/app/api/api_v1/routers/documents.py

```diff
@@ -162,3 +173,84 @@ async def post_association(
         document_association.name,
         document_association.type,
     )
+
+
+@documents_router.post(
+    "/document-relationship", response_model=RelationshipEntityResponse, status_code=201
+)
+async def post_relationship(
+    request: Request,
+    relationship: RelationshipCreateRequest,
+    db=Depends(get_db),
+    current_user=Depends(get_current_active_superuser),
+):
+    """Create a relationship"""
+    return create_relationship(
+        db,
+        relationship.name,
+        relationship.type,
+        relationship.description,
+    )
+
+
+@documents_router.get("/document-relationship", response_model=RelationshipGetResponse)
+async def get_relationship(
+    request: Request,
+    db=Depends(get_db),
+    current_user=Depends(get_current_active_superuser),
+):
+    """Get all relationsgips"""
+    return get_relationships(db)
+
+
+@documents_router.get(
+    "/document-relationship/{relationship_id}",
```

Comment:
nit: until this point we have used single word plurals for the root of URLs... e.g. `relationships`